src/extension.ts

```diff
@@ -525,10 +535,50 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
     outputChannel.show(true);
   }
 
+  const languageServerExt = await languageServer();
+  if (!languageServerExt) {
```

Comment:
I suggest you add an `outputChannel.appendLine` call in this block to notify user that the language server wasn't found.